src/main/resources/SnakemakeBundle.properties

```diff
@@ -42,6 +42,7 @@ INSP.NAME.only.n.parameters.in.section=Don''t use more than {0} lambda parameter
 INSP.NAME.wildcards.first.parameter='wildcards' has to be the first lambda parameter.
 INSP.NAME.wildcards.first.parameter.preferable=Snakemake documentation suggests it's preferable to name the first parameter 'wildcards'.
 INSP.NAME.non.wildcards.param.first.parameter=''{0}'' cannot be the first lambda parameter in ''{1}'' section.
+INSP.NAME.callables.not.allowed.in.section=Callables are not allowed in ''{0}'' section.
```

Comment:
Let's replace `callables are not allowed` with smth more user-friendly.  E.g. smth like 'String value is expected ..' or `Function object cannot be a value in ...` or etc

Reply:
done